code/modules/mob/living/simple_animal/friendly/capybara.dm

```diff
@@ -0,0 +1,43 @@
+/mob/living/simple_animal/capybara
+	name = "capybara"
+	real_name = "capybara"
+	desc = "A brown capybara, or a \"coconut dog\", being relaxed as usual."
+	icon = 'icons/mob/npc/coconut_dog.dmi'
```

Comment:
If the typepath is capybara, it would be optimal for the icon file to be called capybara too